QuizMakerAngular/src/styles.css

```diff
@@ -1 +1,64 @@
 /* You can add global styles to this file, and also import other style files */
+/* You can add global styles to this file, and also import other style files */
+body {
+    margin: 0;
+    padding: 0;
+    font-family: sans-serif;
+}
+
+.container {
+    display: grid;
+    grid-template-columns: 200px 1fr 200px;
+    grid-template-rows: auto 1fr auto;
+    grid-gap: 12px;
+    grid-template-areas:
+    "header header header"
+    "nav content side"
+    "footer footer footer";
+    height: 100vh;
+  }
+
+app-header {
+    color:black;
+    font-size: 14px;
+    background-color: grey;
+    grid-area: header;
+  }
+  
+  app-nav {
+    grid-area: nav;
+    margin-left: 0.9rem;
+    background-color: darkgrey;
+    width: 400px;
+  }
+  
+  app-privacy-policy,
+  app-main {
+    grid-area: content;
+    padding: 25px;
+    width: 1000%;
```

Comment:
Is this meant to be 100%? Because 1000 seems a bit...extreme xD

Reply:
oh , that's supposed to be 100% . I will fix that.